src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1857,6 +1865,12 @@ def load_arguments(self, _):
             "disable_fips_image",
             action="store_true"
         )
+        c.argument(
+            'localdns_config',
+            options_list=['--localdns-config'],
```

Comment:
This is the default option format (handled by the CLI framework), so you can omit it unless you want to add an alternative form, like a shorthand alias

Reply:
Oh I see, thank you! For now, this is good enough then :)